WalletWasabi.Gui/Tabs/WalletManager/GenerateWalletViewModel.cs

```diff
@@ -47,61 +44,23 @@ public GenerateWalletViewModel(WalletManagerViewModel owner) : base("Generate Wa
 
 		private void DoNextCommand()
 		{
-			WalletName = Guard.Correct(WalletName);
-
-			if (!ValidateWalletName(WalletName))
-			{
-				NotificationHelpers.Error("Invalid wallet name.");
-				return;
-			}
-
-			string walletFilePath = Path.Combine(Global.WalletsDir, $"{WalletName}.json");
-
-			if (!TermsAccepted)
-			{
-				NotificationHelpers.Error("Terms are not accepted.");
-			}
-			else if (string.IsNullOrWhiteSpace(WalletName))
-			{
-				NotificationHelpers.Error("Invalid wallet name.");
-			}
-			else if (File.Exists(walletFilePath))
-			{
-				NotificationHelpers.Error("Wallet name is already taken.");
-			}
-			else
+			try
 			{
-				try
+				if (!TermsAccepted)
```

Comment:
Please remove everything legal related from the RPC. It's impractical.

Reply:
That's not in the RPC, that is where it has always been.

Reply:
Use `Hide whitespace changes` to see that it was there already.
https://github.com/zkSNACKs/WalletWasabi/pull/3050/files?utf8=%E2%9C%93&diff=unified&w=1